app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/dummy/DevicePresets.java

```diff
@@ -0,0 +1,65 @@
+package edu.drexel.lapcounter.lapcounter.backend.dummy;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Log;
+
+/**
+ * Here is a list of device presets for the Puck, the cheap tags and the Feather.
+ * One is listed as the "selected" device, this setting is stored in SharedPreferences of the
+ * current workout screen.
+ */
+public class DevicePresets {
+    private static final String PREFS_KEY = "lap_counter_test_devices";
+    private static final String KEY_SELECTED_DEVICE ="lap_counter_selected_test_device";
+
+    private static final String[] MAC_ADDRESSES = new String[]{
```

Comment:
If this is something that will be used in the "final" version, I would recommend putting the mac addresses and names into Device Objects and use the getters.
However, if this is just for testing, then it doesnt matter